test/unit/functionsAsyncAwait.test.js

```diff
@@ -0,0 +1,277 @@
+const {
+  getFilteredsStargazers,
+  getTopFive,
+  sortDataByDate,
+  getSumOfReposStars,
+  getReposMoreThanFiveStars,
+  getLastUpdatedRepos,
+} = require("../../scripts/functionsAsyncAwait.js");
+
+const { data: completeData } = require("../fixtures/completeData.js");
+const { data2 } = require("../fixtures/incompleteData.js");
+
+beforeEach(() => {
+  data = [...completeData];
+});
+
+describe("getReposMoreThanFiveStars", () => {
+  test("returns an array with repos with more than 5 stars", () => {
+    expect(getFilteredsStargazers(data)).toEqual([
+      {
+        name: "repo1",
+        stargazers_count: 10,
+        updated_at: "2020-11-18T12:00:00Z",
+      },
+      {
+        name: "repo5",
+        stargazers_count: 9,
+        updated_at: "2024-11-18T12:00:00Z",
```

Comment:
Add a description to specify the case when this happens
```suggestion
describe("getReposMoreThanFiveStars", () => {
  describe("when an array of valid data is passed",() => {
   test("returns an array with repos with more than 5 stars", () => {
  
```